api/kessel/inventory/v1beta2/subject_reference.proto

```diff
@@ -15,5 +15,5 @@ message SubjectReference {
   // An optional relation which points to a set of Subjects instead of the single Subject.
   // e.g. "members" or "owners" of a group identified in `subject`.
   optional string relation = 1;
-  ObjectReference subject = 2 [(buf.validate.field).required = true];
+  ResourceReference subject = 2 [(buf.validate.field).required = true];
```

Comment:
```suggestion
  ResourceReference resource = 2 [(buf.validate.field).required = true];
```

This gets rid of the awkward `subject.subject` naming inside the schema (subfield same name as parent field). Plus it is a arguably a little more conceptually accurate (and mirrors what SpiceDB does).